go/webpack/x509util.go

```diff
@@ -0,0 +1,100 @@
+package webpack
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"reflect"
+)
+
+// Parses a sequence of x509 certificates from filename.
+func LoadCertificatesInto(filename string, certs *[]*x509.Certificate) error {
```

Comment:
Sure, done. I could also use just `LoadCertificates()`. I had `Into` to distinguish it from a function that I've since removed.